### PR TITLE
RD JRD load pr pointing to Demo

### PR DESCRIPTION
**Pull request by nayeemshaik-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by shivajihmcts — APPROVED** [review]

**Review by shivajihmcts — DISMISSED** [review]

**Review by adusumillipraveen — COMMENTED** [review]
This is not recommended, AAT should always be prod like.

**Review by ishratt — DISMISSED** [review]

**Comment by nayeemshaik-hmcts** [comment]
> This is not recommended, AAT should always be prod like.

changed the PR image to demo environment

**Review by shivajihmcts — APPROVED** [review]

File: apps/rd/rd-judicial-data-load/aat-00.yaml [hunk 1]
```diff
@@ -10,3 +10,5 @@ spec:
         DELETE_ORG: false
         JRD_SIDAM_SEARCH_QUERY: "(roles:judiciary) AND lastModified:>now-116h"
         MAIL_FROM: no-reply@mail-rd-nonprod.platform.hmcts.net
+    java:
```
Comment (timja) [inline apps/rd/rd-judicial-data-load/aat-00.yaml:13]:
```suggestion
```

File: apps/rd/rd-judicial-data-load/aat-00.yaml [hunk 2]
```diff
@@ -10,3 +10,5 @@ spec:
...
+      image: hmctspublic.azurecr.io/rd/rd-judicial-data-load:pr-518
```
Comment (timja) [inline apps/rd/rd-judicial-data-load/aat-00.yaml:14]: you may not deploy PR images to the AAT environment

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/rd/rd-judicial-data-load/aat-00.yaml (hunk 2 above)
Comment (nayeemshaik-hmcts) [reply apps/rd/rd-judicial-data-load/aat-00.yaml:14]: changed the  PR image pointing to demo environment .

File: apps/rd/rd-judicial-data-load/aat-00.yaml (hunk 1 above)
Comment (nayeemshaik-hmcts) [reply apps/rd/rd-judicial-data-load/aat-00.yaml:13]: removed the entry in aat-00.yml file

**Review by timja — COMMENTED** [review]
nothing appears to have changed please check you pushed

**Review by vasthav-patta — APPROVED** [review]

**Review by vilasshelke-hmcts — DISMISSED** [review]

**Comment by nayeemshaik-hmcts** [comment]
> nothing appears to have changed please check you pushed

Pushed the latest changes and its points the PR image to demo

File: apps/rd/rd-judicial-data-load/demo-01.yaml [hunk 3]
```diff
@@ -8,3 +8,5 @@ spec:
       schedule: "15 12 * * *"
       environment:
         DELETE_ORG: true
+    java:
```
Comment (timja) [inline apps/rd/rd-judicial-data-load/demo-01.yaml:11]:
```suggestion
```

File: apps/rd/rd-judicial-data-load/aat-00.yaml [hunk 4]
```diff
@@ -9,4 +9,4 @@ spec:
       environment:
         DELETE_ORG: false
         JRD_SIDAM_SEARCH_QUERY: "(roles:judiciary) AND lastModified:>now-116h"
-        MAIL_FROM: no-reply@mail-rd-nonprod.platform.hmcts.net
+        MAIL_FROM: no-reply@mail-rd-nonprod.platform.hmcts.net
```
Comment (timja) [inline apps/rd/rd-judicial-data-load/aat-00.yaml:12]: undo changes to this file

File: apps/rd/rd-judicial-data-load/demo-01.yaml (hunk 3 above)
Comment (nayeemshaik-hmcts) [reply apps/rd/rd-judicial-data-load/demo-01.yaml]: thanks @timja , removed the java and moved under job

File: apps/rd/rd-judicial-data-load/demo-01.yaml [hunk 5]
```diff
@@ -8,3 +8,4 @@ spec:
       schedule: "15 12 * * *"
       environment:
         DELETE_ORG: true
+      image: hmctspublic.azurecr.io/rd/rd-judicial-data-load:pr-518
```
Comment (timja) [inline apps/rd/rd-judicial-data-load/demo-01.yaml:11]: you need to supply an image policy reference here, see example:
https://github.com/hmcts/cnp-flux-config/blob/e0be11b80c2225453b733ea18688a3b284ca0c0f/apps/rpe/draft-store-service/draft-store-service.yaml#L12

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/rd/rd-judicial-data-load/demo-01.yaml (hunk 5 above)
Comment (nayeemshaik-hmcts) [reply apps/rd/rd-judicial-data-load/demo-01.yaml]: updated the image full path

**Review by timja — APPROVED** [review]

**Review by vasthav-patta — APPROVED** [review]

**Pull request merged**